app/views/brexit_checker/_share_links.html.erb

```diff
@@ -10,27 +7,27 @@
     links: [
       {
         href: "https://www.facebook.com/sharer/sharer.php?u=#{encoded_results_url}",
-        text: hidden_text + "Facebook",
+        text: "Facebook",
         icon: "facebook"
       },
       {
         href: "https://twitter.com/share?url=#{encoded_results_url}",
-        text: hidden_text + "Twitter",
+        text: "Twitter",
         icon: "twitter"
       },
       {
         href: "https://api.whatsapp.com/send?text=#{encoded_results_url}",
-        text: hidden_text + "WhatsApp",
+        text: "WhatsApp",
         icon: "whatsapp"
       },
       {
         href: "mailto:?body=#{encoded_results_url}&subject=#{I18n.t("brexit_checker.results.title")}",
-        text: hidden_text + "Email",
+        text: "Email",
         icon: "email"
       },
       {
         href: "http://www.linkedin.com/shareArticle?url=#{encoded_results_url}",
-        text: hidden_text + "LinkedIn",
+        text: "LinkedIn",
```

Comment:
@danacotoran so this [similar PR](https://github.com/alphagov/government-frontend/pull/1916/files) had a testing file for this element, what are the rules around whether to include this or not?

Reply:
This repo already has a [test](https://github.com/alphagov/finder-frontend/blob/ebca596afe84d00fa472cf841f77fa409ff6b47b/spec/features/brexit_checker/question_results_spec.rb#L65) to check for the presence of share links on the results page as well.

Strangely, the test in the other repo checks that the share links have a specific text, whereas this one checks that the `href`s are correct.  Either way, they both check that the links exist.